src/tasks/tyeRunTaskProvider.ts

```diff
@@ -97,9 +97,15 @@ export default class TyeRunCommandTaskProvider extends CommandTaskProvider {
                                         waitForProcessClose: false
                                     };
                                 },
-                                onStarted:
-                                    pid => {
-                                        tyePid = pid;
+                                onStdOut: stdOut => {
+                                        let matchingLogLine = null;
+                                        const tyeAppStartedRegex = new RegExp('started successfully with Pid: [0-9]+');
```

Comment:
Nit: move the `RegEx` constant out of the function.  (I kind of assume the compiler would optimize this anyway, but you never know.)

Reply:
Moved it out of the function!